common/presenters/move-to-meta-list-component.js

```diff
@@ -101,6 +105,18 @@ function moveToMetaListComponent(
         text: filters.formatTime(timeDue),
       },
     },
+    {
+      key: {
+        text: i18n.t('fields::prison_transfer_type.label'),
+      },
+      value: {
+        text:
```

Comment:
Finding the legibility of this really hard - it's not the line length it's nested ternaries and the double check plus the concatenation (for the nation). Struggling with the one at line 67 too to be honest. Maybe calculate the value before setting the property?